client/web/compose/src/components/ModuleFields/Editor/Bool.vue

```diff
@@ -1,34 +1,46 @@
 <template>
   <b-form-group
-    label-class="d-flex align-items-center text-primary p-0"
+    :label-cols-sm="options.horizontalFieldLayoutEnabled && '5'"
+    :label-cols-lg="options.horizontalFieldLayoutEnabled && '4'"
+    :content-cols="options.horizontalFieldLayoutEnabled"
   >
     <template
-      v-if="field.options.switch"
+      v-if="field.options.switch || options.horizontalFieldLayoutEnabled"
       #label
     >
-      {{ label }}
+      <div
+        v-if="!valueOnly"
+        class="d-flex align-items-center text-primary p-0"
+      >
+        <span
+          class="d-inline-block text-truncate mw-100 py-1"
+          :title="label"
+        >
+          {{ label }}
+        </span>
 
-      <hint
-        :id="field.fieldID"
-        :text="hint"
-      />
-    </template>
+        <hint
+          :id="field.fieldID"
+          :text="hint"
+        />
 
-    <div
-      v-if="field.options.switch"
-      class="small text-muted"
-      :class="{ 'mb-1': description }"
-    >
-      {{ description }}
-    </div>
+        <slot name="tools" />
+      </div>
+      <div
+        class="small text-muted"
+        :class="{ 'mb-1': description }"
+      >
+        {{ description }}
+      </div>
+    </template>
 
     <c-input-checkbox
       v-model="value"
       :switch="field.options.switch"
       :labels="field.options.switch ? checkboxLabel : {}"
     >
       <div
-        v-if="!field.options.switch"
+        v-if="!field.options.switch && !options.horizontalFieldLayoutEnabled"
```

Comment:
You dont need the extra check since the only thing that matters is the first condition